_rules/html-page-non-empty-title-2779a5.md

```diff
@@ -39,7 +39,7 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to the root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element.
+This rule applies to the root element of the [web page](#web-page-html), if it is an `html` element.
```

Comment:
I think the link needs to be defined in the definition list in order to be picked up when building the page 🤔 

* At the bottom of the file, add the definition `[web page]: #web-page-html 'Definition of HTML web page'` (next to the existing definition, on a separate line).
* Here, simply reference it with `[web page][]`.

Reply:
Change has been made. Please review and let me know if I missed it. Thank you.